src/test/java/utils/DefectSeverity.java

```diff
@@ -0,0 +1,16 @@
+package utils;
+
+import lombok.Getter;
+
+@Getter
+public enum DefectSeverity {
```

Comment:
Please move this enum to 'enums' package that you already have in the project structure